### PR TITLE
10035 impr sign trans

### PR DESCRIPTION
**Pull request by endulab**
### What does the PR do

Compute deployment fees in async way.
Improve showing errors when estimating fees.

Related status-go pr: https://github.com/status-im/status-go/pull/3369

### Affected areas

Minting token

### Screenshot of functionality (including design for comparison)

https://user-images.githubusercontent.com/61889657/229502106-106158e2-e456-48f2-b992-27caad6add6a.mov

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
Looks good to me, nice job

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 1]
```diff
@@ -322,17 +322,27 @@ StatusSectionLayout {
...
-                    function onDeployFeeUpdated(value) {
-                        // TODO better error handling
-                        if (value === "-") {
-                            mintPanel.isFeeLoading = true
-                        } else {
+                    function onDeployFeeUpdated(ethFee, fiatFee, errorCode) {
+                        if (errorCode === Constants.ComputeFeeErrorCode.Success || errorCode === Constants.ComputeFeeErrorCode.Balance) {
+                            let ethRounded = (ethFee).toFixed(4)
+                            let fiatRounded = (fiatFee).toFixed(2)
+                            let valueStr = ethRounded + "ETH ("+ fiatRounded + rootStore.currentCurrency +")"
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:331]: This deserves some proper formatting :)

Comment (endulab) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:331]: What do you mean ?

**Review by borismelnik — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml (hunk 1 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:331]: or spaces between math operations? :)

Comment (endulab) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml]: Ok, I fixed this and use LocaleUtils.currencyAmountToLocaleString now.

Comment (caybro) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml]: Great, thanks!

**Review by caybro — APPROVED** [review]

**Pull request merged**